woocommerce/woocommerce-ios
# Blaze Optimization: UI for Blaze flow entry on Product editing form screen

Part of: https://github.com/woocommerce/woocommerce-ios/issues/10961



## Description


This PR does two things:
1. It removes the Blaze option from the Product editing screen's More menu.
2. It adds a "Promote with Blaze" button on the Product editing screen, above "Price".

Known issue: no related unit test yet especially related to the "Promote with Blaze" button. This will be added in the next PR for functionalities.

## Testing instructions

1. Start with a site that is eligible for Blaze,
2. Go to Product details, tap a Product,
4. Ensure that a button that says "Promote with Blaze" appears below "Description" and above "Price".

## Screenshots


| Dark mode | Light mode |
|-|-|
| ![Simulator Screenshot - iPhone 14 Pro - 2023-10-23 at 20 26 42](https://github.com/woocommerce/woocommerce-ios/assets/266376/57e56ff4-3636-4e99-ad1a-9cbd8cd3046f) | ![Simulator Screenshot - iPhone 14 Pro - 2023-10-23 at 20 24 18](https://github.com/woocommerce/woocommerce-ios/assets/266376/1647dffa-69d7-49dc-a7b4-c05fbc814f17) |
